### DISCUSSION TITLE
How to debug engine.vcxproj (eg: core.cpp)

### DISCUSSION THREAD

**Discussion by yuvnith** [wixtoolset/issues#8705] [Questions]
I am trying to debug the engine. vcxproj to understand the detect, plan, etc. workflows. 
When I attempted to attach the setup to the burn.sln Visual Studio instance, the symbols were loaded and the breakpoints were also active. However, the breakpoints were not hit.

Could someone assist with this core.cpp debugging?

Thanks in advance.

**Comment by bevanweiss** [comment]
Have you tried just putting an Assert in the code that will be hit?
That should allow you to debug into that point.

**Reply by yuvnith** [reply]
Hi, Thanks for the comment. No, I was not able to debug from this point
